cmd/roy/data/wikidata/wikidata-test-definitions

```diff
@@ -1,6 +1,6 @@
 {
   "head" : {
-    "vars" : [ "uri", "uriLabel", "puid", "extension", "mimetype", "encodingLabel", "referenceLabel", "date", "relativityLabel", "offset", "sig" ]
+    "vars" : [ "uri", "uriLabel", "puid", "extension", "mimetype", "encoding", "referenceLabel", "date", "relativity", "offset", "sig" ]
```

Comment:
Removing label from encoding and relativity fields will give us the IRIs for those values instead. The tests are updated here to verify this change. A few new entries have been added to demonstrate Siegfried results in other encodngs. 

Mappings to pay attention to below: 

* IRI ending Q82828 - Hexadecimal.
* IRI ending Q35436009 - beginning of file.
* IRI ending Q1148480 - end of file.

Reply:
NB. I have added annotations to this file to help with any understanding in future, but I am not wedded to those annotations moving forward.